src/components/Worker.tsx

```diff
@@ -0,0 +1,111 @@
+interface WorkerProps {workerInfo: {id: number, firstName: string, lastName:string,
+     title:string, country: string, city:string, birthDate:Date, imageUrl:string, time: string}}
+
+const Worker = ({workerInfo}: WorkerProps) => {
+    const age: number = new Date().getFullYear() - workerInfo.birthDate.getFullYear();
+    console.log(workerInfo.birthDate);
+    return (
+        <>
+            <div className="col-md-4">
+                <div className="team text-center">
+                    <img src={workerInfo.imageUrl} alt="Worker Image" className="avatar" />
+                    <div className="title">
+                        <h4>{workerInfo.firstName + " " + workerInfo.lastName}</h4>
+                        <h5 className="muted regular">{workerInfo.title}</h5>
+                        <h5 className="muted regular">{workerInfo.city + ", " + workerInfo.country}</h5>
+                    </div>
+                    <button
+                        data-toggle="modal"
+                        data-target="#modal1"
+                        className="btn btn-blue-fill"
+                    >
+                        About Me
+                    </button>
+                </div>
+            </div>
+            <div id="modals">
```

Comment:
The modal looks fine (aside from the grey text color). Change the css of the card itself to make it look uniform and better.